resources/datadog-integrations/software.rb.erb

```diff
@@ -0,0 +1,24 @@
+name "dd-check-<%= name %>-software"
+default_version "0.0.1"
+
+require 'json'
+require_relative "<%= PROJECT_DIR %>/resources/datadog-integrations/validate_manifest"
+integration_source_folder = '/<%= integrations_repo %>/<%= name %>'
+integration_agent_folder = "#{install_dir}/3rd-party/<%= name %>"
+
+build do
+  # validate manifest.json
+  manifest = JSON.parse(File.read("#{integration_source_folder}/manifest.json"))
+  validate_manifest(manifest)
+
+  # copy files
+  mkdir "#{integration_agent_folder}"
+  copy "#{integration_source_folder}/conf.yaml.example", "/etc/dd-agent/conf.d/<%= name %>.yaml.example"
+  command "echo \"#{integration_agent_folder}/lib\" > #{install_dir}/embedded/lib/python2.7/site-packages/#{name}.pth"
+  for file in ["check.py", "manifest.json", "requirements.txt", "README.md"]
+    copy "#{integration_source_folder}/#{file}", "#{install_dir}/3rd-party/<%= name %>/#{file}"
+  end
+
+  # install requirements
+  command "#{install_dir}/embedded/bin/pip install -I --target=#{install_dir}/3rd-party/<%= name %>/lib -r #{integration_source_folder}/requirements.txt"
```

Comment:
nitpick: let's use `#{integration_agent_folder}` instead of `#{install_dir}/3rd-party/<%= name %>` here